### DISCUSSION TITLE
2022-04-05-utterances-blog-comments

### DISCUSSION THREAD

**Discussion by giscus** [mauris/site-content#11] [General]
# 2022-04-05-utterances-blog-comments

https://mauris.github.io/blog/posts/2022-03-31-nextjs-typescript-monorepo

**Comment by mauris** [comment]
thanks @laymonage! Looks great here on giscus too, and it seems like there's more features too, like reactions for the page/post itself. 

It was also easy to integrate / add into the Next.js blog.

**Reply by laymonage** [reply]
Glad you like it!
